proof-systems-shim/src/lib.rs

```diff
@@ -14,6 +14,7 @@
 //! ```
 //!
 
+pub use mina_curves;
```

Comment:
You don't need to change this crate, Fp is re-exported from mina_hasher

Reply:
fixed